src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -90,13 +90,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Include="/OPT:ICF" />
     </ItemGroup>
 
-    <Exec Command="where /Q $(CppCompiler) &amp;&amp; where /Q $(CppLinker)" IgnoreExitCode="true">
-      <Output TaskParameter="ExitCode" PropertyName="_WhereCppTools" />
-    </Exec>
-
-    <Message Condition="'$(_WhereCppTools)' != '0'" Text="Tools '$(CppCompiler)' and '$(CppLinker)' not found on PATH. Attempting to autodetect." />
-
-    <Exec Condition="'$(_WhereCppTools)' != '0'" Command="&quot;$(MSBuildThisFileDirectory)findvcvarsall.bat&quot; $(TargetArchitecture)"
+    <Exec Command="&quot;$(MSBuildThisFileDirectory)findvcvarsall.bat&quot; $(TargetArchitecture)"
```

Comment:
We should have a property to disable the auto-detection so that one can explicitly provide the environment if needed.

This is useful when one has multiple VS versions installed on the box and you want to use a different VS version that the one picked up by the autodetection.